src/registry2.0/snapshot.c

```diff
@@ -42,19 +42,6 @@
 #include "registry.h"
 #include "util.h"
 
-/**
- * Converts a command name into a `reg_snapshot`.
- *
- * @param [in] interp  Tcl interpreter to check within
- * @param [in] name    name of snapshot to get
- * @param [out] errPtr description of error if the snapshot can't be found
- * @return             an snapshot, or NULL if one couldn't be found
- * @see get_object
- */
-static reg_snapshot* get_snapshot(Tcl_Interp* interp, char* name, reg_error* errPtr) {
```

Comment:
I kept this function intentionally to remind me that the identity to identify/fetch a snapshot is different from what we use for a port/portgroup/file (where we use Tcl_HashEntry* hash and Tcl_GetIndexFromObj).

So, is that fine? At present, it just uses sqlite-id and I treat id of a snapshot as a property for fetching it, though this is not what we do for port/portgroup/file etc.